pkg/services/searchV2/auth.go

```diff
@@ -72,7 +76,7 @@ func (a *simpleSQLAuthService) GetDashboardReadFilter(user *user.SignedInUser) (
 		uids[rows[i].UID] = true
 	}
 
-	return func(uid string) bool {
+	return func(_ entityKind, _, uid string) bool {
```

Comment:
I think this one is wrong, uid is the second argument?

Reply:
Ouph, good spot! Corrected it.